state/application_test.go

```diff
@@ -222,80 +222,6 @@ func (s *ApplicationSuite) TestSetCharmUpdatesBindings(c *gc.C) {
 	})
 }
 
-func (s *ApplicationSuite) TestSetCharmWithWeirdlyNamedEndpoints(c *gc.C) {
```

Comment:
will we be changing things later to escape/unescape? if yes, skip rather than delete?